webui/react/src/components/FilterForm/TableFilter.tsx

```diff
@@ -49,7 +50,7 @@ const TableFilter = ({
         placement="bottomLeft"
         trigger="click"
         onOpenChange={onIsOpenFilterChange}>
-        <Button icon={<FilterOutlined />}>
+        <Button icon={<Icon name="filter" title="filter button" />}>
```

Comment:
fix: these don't need to say "<whatever> button". The button already announces itself as such.